docs/kubectl_rolling-update.md

```diff
@@ -36,6 +36,7 @@ $ kubectl rolling-update frontend --image=image:v2
 ### Options
 
 ```
+      --container-name="": Container name which will have its image upgraded. Only relevant when --image is specified, ignored otherwise. Required when using --image on a multi-container pod
```

Comment:
Please be consistent with other commands that have container-name flags. For instance:
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/exec.go#L68